test/chain-specific-spokepools/Polygon_SpokePool.ts

```diff
@@ -280,130 +273,6 @@ describe("Polygon Spoke Pool", function () {
       .reverted;
   });
 
-  it("Must be EOA to execute relayer refund leaf with amountToReturn > 0", async function () {
```

Comment:
This was only recently added and wasn't directly related to USS/non-USS updates - confirmed that we want to remove it?

Reply:
Yes because we have the same test for the USS execute function, right?